src/scripts/glue_jobs/dynamic_frame_mapping.py

```diff
@@ -0,0 +1,47 @@
+"""
+DynamicFrame : A Dynamic Frame is similar to an Apache Spark dataframe, which is a data abstraction used to
+organize data into rows and columns, except that each record is self-describing so no schema is required initially.
+see link: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-crawler-pyspark-extensions-dynamic-frame.html
+
+DynamicRecord : a DynamicRecord represents a logical record within a DynamicFrame. It is like a row in a Spark
+DataFrame. see link: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-crawler-pyspark-extensions-dynamic-frame.html
+
+Glue reads data from s3 source and creates a dynamic frame. Glue computes the schema for dynamic frame. Apply mapping
+methods changes the schema, e.g, field name and datatype. """
+
+from awsglue.transforms import *
+
+def file_download_record_mapping(input_frame):
```

Comment:
I would define an object that defines "named" mappings that contain the mapping configuration, for example:

mapping_refs = {
  "file_downloads": [...],
  "file_uploads": [..]
}

Then we can have a utility function to apply a given mapping:

def apply_mapping(input_frame, mapping_name):
  return input_frame.apply_mapping(mapping_refs[mapping_name])

We would also need to refactor the other scripts to use this.